tutorials/plot_stats_cluster_1samp_test_time_frequency.py

```diff
@@ -84,8 +84,9 @@
 # Compute statistic
 # -----------------
 threshold = 2.5
+n_permutations = 100  # Warning: 100 is too small please use a bigger number
```

Comment:
Improve sentence.

Something like:

# Warning: 100 is too small for real-world analysis.